lib/rubocop/cop/layout/space_before_block_braces.rb

```diff
@@ -110,6 +118,15 @@ def style_for_empty_braces
           end
         end
 
+        def conflict_with_block_delimiters?
+          block_delimiters_style == 'line_count_based' &&
+            style == :no_space && node.multiline?
```

Comment:
@koic 
With rubocop-0.78.0 I got the below error. Do you have any idea?
```ruby
An error occurred while Layout/SpaceBeforeBlockBraces cop was inspecting /webapp/spec/my_spec.rb:697:6.
undefined local variable or method `node' for #<RuboCop::Cop::Layout::SpaceBeforeBlockBraces:0x0000560a675e42d8>
/usr/local/bundle/gems/rubocop-0.78.0/lib/rubocop/cop/layout/space_before_block_braces.rb:123:in `conflict_with_block_delimiters?'
```

Reply:
@johnny-miyake Thanks for your feedback. Can you open the issue with the reproduction code?
https://github.com/rubocop-hq/rubocop/issues

Reply:
@koic Sure. I opened the issue here: https://github.com/rubocop-hq/rubocop/issues/7590